Dockerfile

```diff
@@ -76,9 +76,6 @@ RUN git clone --depth=1 https://github.com/DataDog/datadog-agent.git /build/data
 
 WORKDIR /build/datadog-agent
 
-COPY fix-ebpf-for-alpine3.13.patch ./
-RUN patch -p1 < fix-ebpf-for-alpine3.13.patch
```

Comment:
DataDog switched to use upstream gobpf and upstream gobpf now supports bcc v0.17.0.
https://github.com/DataDog/datadog-agent/pull/9580
https://github.com/iovisor/gobpf/commit/dcacba608f5153b453bfbe30a2a915e8e867d468